platform/ios/MGLMultiPoint.mm

```diff
@@ -0,0 +1,103 @@
+#import "MGLMultiPoint.h"
+#import "MGLTypes.h"
+
+#import <mbgl/util/geo.hpp>
+
+@implementation MGLMultiPoint
+{
+    CLLocationCoordinate2D *_coords;
```

Comment:
Never mind, the annotation doesn’t have mutable length, and this is consistent with `-points` in MapKit.